Libplanet/Blocks/Block.cs

```diff
@@ -390,6 +390,17 @@ internal void Validate(DateTimeOffset currentTime)
                 );
             }
 
+            var hasTxsOfSameSigner = Transactions
+                .GroupBy(tx => tx.Signer)
+                .Any(s => s.Count() > 1);
+
+            if (hasTxsOfSameSigner)
+            {
+                throw new MultipleTxsOfSameSignerException(
+                    "Only one transaction per signer " +
+                    "can be stored in the block.");
```

Comment:
How about this: “A signer is allowed to have only one transaction at most in a block.”